static/js/ui_init.js

```diff
@@ -612,9 +612,9 @@ export function initialize_everything() {
     about_zulip.initialize();
 
     // All overlays must be initialized before hashchange.js
+    activity.initialize();
     hashchange.initialize();
```

Comment:
We need to move this init around because when activity initializes, it defines `get_filter_text`, which we use to draw the buddy_list, which happens (again, since activity just drew it) via hashchange > narrow > narrow.activate > activity.redraw.

Switching these around doesn't break anything, but leads to the warning "get_filter_text() is called before initialization".

Do we need a comment explaining this?